packages/icon/src/Icon.js

```diff
@@ -2,26 +2,62 @@ import React from 'react';
 import classNames from 'classnames';
 
 type Props = {
-  /** Font Awesome icon name */
+  /** Streamline icon name */
   name: string,
+  /** Add optional styling */
   style?: object,
+  /** Add optional CSS class */
   className?: string,
-  /** optional ariaLabel: check WCAG compliance for extra info when a label is needed. */
+  /** Add optional ariaLabel: check WCAG compliance for extra info when a label is needed. */
   ariaLabel?: string,
-  /** optional on click */
+  /** Add optional onClick */
   onClick?: (e: object) => void,
+  /** Provide your own SVG icon */
+  src?: string,
   /** Qa id */
   qa?: string,
 };
 
-const Icon = ({ name, style, className, ariaLabel, onClick, qa }: Props) => {
-  const iconClass = classNames(className, `fa fa-${name} ${name}`);
+let isFetching = false;
+
+const Icon = ({ name, style, className, ariaLabel, onClick, src, qa }: Props) => {
+
+  /**
+   * Inject SVG icons from core branding into page
+   */
+  const fetchAntwerpIcons = async () => {
+    try {
+      const xlinkHref = 'https://cdn.antwerpen.be/core_branding_scss/5.0.0-beta.5/assets/images/ai.svg';
+      const response = await fetch(xlinkHref);
+      const svgText = await response.text();
+      const svgWrapper = document.createElement('svg');
+
+      svgWrapper.id = 'aiSvg';
+      svgWrapper.innerHTML = svgText;
+      if(!document.getElementById('aiSvg')) {
+        document.body.appendChild(svgWrapper);
+      }
+    } catch(err) {
+      // Do nothing, just make sure it is executed again
```

Comment:
call fetchAntwerpIcons() again, maybe keep track of a retry count? use a setTimeout() of 5 seconds?...